ApplicationInstallationInUbuntu1804.md

```diff
@@ -0,0 +1,70 @@
+# Application Installation in Ubuntu 18.04
+
+## Step 1: Create a virtual env
+
+`sudo apt-get update`
+
+`sudo apt-get install python3-venv`
+
+`python3 -m venv encoded-venv`
+
+`source encoded-venv/bin/activate`
+
+## Step 2: install dependencies:
+
+Some tools may not be preinstalled in Unbuntu.
+
+`sudo apt install awscli  unattended-upgrades apache2 apache2-dev apache2-utils ssl-cert   libapache2-mod-wsgi-py3 libjpeg8-dev zlib1g-dev libpq-dev apt-transport-https daemontools  build-essential libbz2-dev libdb-dev libffi-dev libgdbm-dev liblzma-dev libncurses5-dev libnss3-dev  libreadline-dev libssl-dev libsqlite3-dev python3-dev zlib1g-dev redis-server lzop nginx libarchive-tools bsdtar python3-setuptools -y`
+
+
+## Step 3: clone git
+
+`sudo apt-get install git`.
+
+`git clone https://github.com/utsw-bicf/pandiseased`.
+
+Go to your project folder and checkout the branch you want if not master branch.
+
+## Step 4: run shell script
+
+Find user name using `whoami`
+
+Find home directory using `echo "$HOME"`
+
+Change the user name and home directory to yours in the shell script install-devservers.sh
+
+`sh install-devservers.sh`
+
+`source ~/.bashrc`
+
+## Step 5: build application
+
+`make clean && buildout bootstrap && bin/buildout`
```

Comment:
Shows error with clean buildout step:
![image](https://user-images.githubusercontent.com/38848214/102108834-054cc380-3df9-11eb-9cd5-23fbf5fdc3bf.png)
![image](https://user-images.githubusercontent.com/38848214/102108873-0ed62b80-3df9-11eb-9505-aa1c8e52cf7c.png)
Try to reinstall boto3:
cd ~/encoded-venv/bin/python3 
pip install awscli --upgrade
pip uninstall botocore
pip uninstall boto3
pip install botocore
pip install boto3
(Refer: https://stackoverflow.com/questions/51911075/how-to-check-awscli-and-compatible-botocore-package-is-installed)
>>rerun clean buildout, can pass the buildout, but showed some error:
![image](https://user-images.githubusercontent.com/38848214/102109701-f61a4580-3df9-11eb-9b37-01b355a4e893.png)
Run server failed:
![image](https://user-images.githubusercontent.com/38848214/102109756-04686180-3dfa-11eb-90c6-4d5d63ece700.png)